src/configuration/testcafe-configuration.ts

```diff
@@ -108,12 +108,12 @@ export default class TestCafeConfiguration extends Configuration {
         await this.asyncMergeOptions(options);
     }
 
-    public async asyncMergeOptions (options?: object): Promise<void> {
+    public async asyncMergeOptions (options?: any): Promise<void> {
```

Comment:
Usually we try to avoid using `any`. Is it possible to avoid `any` in this case. If not, then it's ok